core/src/components/input/input.ionic.scss

```diff
@@ -31,3 +31,13 @@
 .input-bottom .error-text {
   color: var(--highlight-color-invalid);
 }
+
+// Input Hover
+// ----------------------------------------------------------------
+
+@media (any-hover: hover) {
+  :host(:hover) {
+    // TODO(FW-6112): Verify the design token is correct once it's available and remove the hardcoded value.
```

Comment:
Design tokens are available in the parent branch `ROU-4848`. Want to take care of it now or handle that later?

Reply:
Updated in https://github.com/ionic-team/ionic-framework/pull/29313/commits/aeebd8665c5742d59e62c626829ed31e41463d92